tests/conftest.py

```diff
@@ -59,9 +59,10 @@ def _is_valid(config):
     # remove invalid configs
     configs = [c for c in configs if _is_valid(c)]
 
-    for c in configs:
+    for ind, c in enumerate(configs):
         config = dict(c)
         config.update(default_args)
+        config["repo_name"] += f"-{ind}"
```

Comment:
Having different project/environment names seemed to fix some issues. Teardown wasn't happening fully reliably, I guess.